src/app/app.module.ts

```diff
@@ -20,6 +20,7 @@ import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { NotFoundComponent } from './components/not-found/not-found.component';
 import { LoginComponent } from './components/operations/login/login.component';
 import { RegisterComponent } from './components/operations/register/register.component';
+import { MyItinirariesComponent } from './components/itinerary/my-itiniraries/my-itiniraries.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7157930)
```suggestion
import { MyItinirariesComponent } from "./components/itinerary/my-itiniraries/my-itiniraries.component";
```